docs/build/isc/v1.1/docs/how-tos/core-contracts/nft/get-nft-metadata.md

```diff
@@ -0,0 +1,233 @@
+---
+description: How to get NFT metadata on L1
+image: /img/logo/WASP_logo_dark.png
+tags:
+  - NFT
+  - EVM
+  - how-to
+---
+import GetNftMetadata from '../../../_partials/how-tos/token/_get-nft-metadata.md';
+import ERC721Admonition from '../../../_admonitions/_ERC721.md';
+
+# Get NFT Metadata
```

Comment:
Should we rename this to:
```suggestion
# Get IRC27 NFT Metadata
```
Would make it more clear that we are not talking about standard ERC721 or similar here